Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:15.2-alpine3.10
```

Comment:
Not sure how keen I am on supporting a docker build right now when the project isn't really in a state to be used by anyone other then developers.